changelog/+e038e218.fixed.md

```diff
@@ -0,0 +1 @@
+Properly clear references to old branches and schema objects from the registry when deleting branches.
```

Comment:
Is this changelog file name expected?

Reply:
I think it does that when it's not tied to an issue so that it sources it from the git commit hash instead. Is that correct @lykinsbd?

Reply:
Correct, the `towncrier create` process will use a hash for the file name if one is not provided, and it's not tied to a specific issue/PR number.

Reply:
Ok, thanks. I don't know if we need to be certain of that. It's probably something related at least.